app/views/comments/new.html.erb

```diff
@@ -0,0 +1,7 @@
+<h1>New Comment</h1>
+
```

Comment:
- [ ] Good job so far but unfortunately I am not able to see the form because the page is blank :cry:  Please make sure to see the forms created  as you can see on the given [wireframes](https://raw.githubusercontent.com/microverseinc/curriculum-rails/main/blog-app/images/blog_user_all_posts.png?token=ABIFLA3SVPWT6MXQ3C2OGFLBHIHES) as required for this project :+1:

Reply:
@Sajeelzafar Please this is  your app right now ![Screenshot_20230204_193817](https://user-images.githubusercontent.com/6615394/216786860-4e0fa6a2-421c-49c5-bdfe-d637caeb9054.png)